magento/magento2
# add more countries to "optional_zip_countries"

### Description (*)
Adding following countries to the optional_zip_countries default configuration as they do not need zip codes:

* bahamas (BS): https://worldpostalcode.com/bahamas/
* lebanon (LB): https://worldpostalcode.com/lebanon/
* uae (AE): http://www.dubaifaqs.com/zip-code-addresses-dubai-uae.php#:~:text=There%20isn't%20anywhere%20in,12345%2C%20Dubai%2C%20UAE%22.


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. login into admin
2. go to stores -> configuration -> general -> general -> country options
3. check if the according countries (BS, AE, LB) are set as optional for zip codes

### Questions or comments


### Contribution checklist (*)
